app/main.py

```diff
@@ -1 +1,8 @@
-# write your code here
+def copy_file(comand: str) -> None:
+    _, old_file, new_file = comand.split()
```

Comment:
```suggestion
    command, old_file, new_file = comand.split()
```
check that command equals to "cp"